front/pages/api/v1/w/[wId]/assistant/conversation/[cId]/sub.ts

```diff
@@ -0,0 +1,87 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { getConversationEvents } from "@app/lib/api/assistant/pubsub";
+import { Authenticator, getAPIKey } from "@app/lib/auth";
+import { ReturnedAPIErrorType } from "@app/lib/error";
+import { Conversation } from "@app/lib/models";
+import { apiError, withLogging } from "@app/logger/withlogging";
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<ReturnedAPIErrorType>
+): Promise<void> {
+  const keyRes = await getAPIKey(req);
+  if (keyRes.isErr()) {
+    return apiError(req, res, keyRes.error);
+  }
+
+  if (!keyRes.value.isSystem) {
+    return apiError(req, res, {
+      status_code: 400,
+      api_error: {
+        type: "invalid_request_error",
+        message:
+          "The Assitant API is only accessible by system API Key. Ping us at team@dust.tt if you want access to it.",
+      },
+    });
+  }
+  const { keyWorkspaceId } = await Authenticator.fromKey(
+    keyRes.value,
+    req.query.wId as string
+  );
+
+  if (keyWorkspaceId !== req.query.wId) {
+    return apiError(req, res, {
+      status_code: 400,
+      api_error: {
+        type: "invalid_request_error",
+        message: "The Assistant API is only available on your own workspace.",
+      },
+    });
+  }
+
+  const conv = await Conversation.findOne({
```

Comment:
Let's remove the Conversation loading for now

Reply:
What happens if we call getConversationEvents on a bogus sId?

Reply:
it will block until the timeout has been reached out and return 0 event.